cdap-watchdog-api/src/main/java/co/cask/cdap/api/metrics/MetricStore.java

```diff
@@ -43,8 +36,7 @@
    * @return time series that satisfy the query
    * @throws Exception
    */
-  // todo: expose metric-specific APIs instead of Cube ones
```

Comment:
So I don't have a lot of context here, but why did you decide to expose "metrics" and not "Cubes" ? What's the difference?